cli/tools/info.rs

```diff
@@ -527,15 +527,15 @@ impl<'a> GraphDisplayContext<'a> {
 
   fn build_module_info(&mut self, module: &Module, type_dep: bool) -> TreeNode {
     enum PackageOrSpecifier {
-      Package(NpmResolutionPackage),
+      Package(Box<NpmResolutionPackage>),
```

Comment:
I assume this is now flagged by linter because of enum variant size diff?

Reply:
yes